conda-store-server/conda_store_server/server/app.py

```diff
@@ -254,6 +254,14 @@ async def http_exception_handler(request, exc):
                 status_code=exc.status_code,
             )
 
+        # Prints exceptions to the terminal
+        # https://fastapi.tiangolo.com/tutorial/handling-errors/#re-use-fastapis-exception-handlers
+        # https://github.com/tiangolo/fastapi/issues/1241
+        @app.exception_handler(Exception)
+        async def exception_handler(request, exc):
+            print(exc)
+            return await http_exception_handler(request, exc)
```

Comment:
Added this to print an exception to the terminal when an invalid `role_mappings_version` value is used. Previously, nothing was printed except "Invalid server error".